### PR TITLE
Loosened Rubydora dependency for 2.x.

### PR DESCRIPTION
**Pull request by dchandekstark**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,3 +11,4 @@ end
 gem 'jruby-openssl', :platform=> :jruby
 gem 'activemodel', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
 gem 'linkeddata', '~> 1.99'
+gem 'rake', '< 12'
```
Comment (jcoyne) [inline Gemfile]: why are you changing this dependency?

Comment (dchandekstark) [inline Gemfile]: I guess rspec 2.99 tasks not compatible with rake 12.

```
$ rake ci
rake aborted!
NoMethodError: undefined method `last_comment' for #<Rake::Application:0x007f96fb031fa8>
/Users/dc/.rvm/gems/ruby-2.3.1@active-fedora/gems/rspec-core-2.99.2/lib/rspec/core/rake_task.rb:143:in `initialize'
lib/tasks/active_fedora_dev.rake:28:in `new'
lib/tasks/active_fedora_dev.rake:28:in `block in <top (required)>'
lib/tasks/active_fedora_dev.rake:7:in `<top (required)>'
/Users/dc/github/projecthydra/active_fedora/Rakefile:10:in `load'
/Users/dc/github/projecthydra/active_fedora/Rakefile:10:in `<top (required)>'
```

Comment (jcoyne) [inline Gemfile]: Why not upgrade rspec to rspec 3?

Comment (dchandekstark) [inline Gemfile]: Upgrading to RSpec 3.5 creates 72 test failures.

Comment (jcoyne) [inline Gemfile]: Have you tried using http://yujinakayama.me/transpec/ ?

Comment (jcoyne) [inline Gemfile]: @dchandekstark Chris informs me that he already ran transpec on AF 8.  Is there some reason you are using AF 7?

Comment (dchandekstark) [inline Gemfile]: Well, I wanted to use AF 8, but discovered that the bump to ActiveTriples 0.4.x was a problem, so was trying to go this route.

Comment (dchandekstark) [inline Gemfile]: Just pinning rake to < 12 seemed the easiest solution all around given that I just want to be able to use Rubydora 2.0.0.

Comment (mjgiarlo) [inline Gemfile]: That satisfies me. @jcoyne ?

**Pull request merged**